src/compenents/todoItems.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from "react";
+
+const TodoItems = () => {
+  const [todoItems, setTodosItems] = useState([
+    {
+        id: 1,
+        text: 'Learn React',
+        completed: false
+    },
+    {
+        id: 2,
+        text: 'Learn Redux',
+        completed: false
+    },
+    {
+        id: 3,
+        text: 'Learn Node',
+        completed: false
+    },
+    {
+        id: 4,
+        text: 'Learn Express',
+        completed: false
+    }
+  ]);
+
+  const handleChange = (id) => {
+    setTodosItems((prevState) =>
+      prevState.map((todo) => {
+        if (todo.id === id) {
+          return {
+            ...todo,
+            completed: !todo.completed,
+          };
+        }
+        return todo;
+      })
+    );
+  };
```

Comment:
When clicked on the checkbox item, it's not getting marked as complete. When marked as complete, the text should be marked as deleted one.